src/components/CustomPressable/index.js

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 import {Pressable} from 'react-native';
 
-import {COLORS} from '../../utils/theme';
+import {colors} from '../../utils/colors';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement has been updated to use the `colors` module instead of the `COLORS` module. Ensure that the `colors` module exists and exports the same properties as the `COLORS` module. If not, this could lead to undefined values being used in the application.
```diff
- import {COLORS} from '../../utils/theme';
+ import {colors} from '../../utils/colors';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->